packages/victory-core/src/victory-util/wrapper.js

```diff
@@ -20,6 +20,7 @@ import Events from "./events";
 import Collection from "./collection";
 import Helpers from "./helpers";
 import Scale from "./scale";
+import { getFormattedData } from "./../../../victory-histogram/src/helper-methods";
```

Comment:
So I'm a bit unsure about how to best handle this. I could pull this function out into victory-core, but I would have to pull out all of the functions related to this, so that seems not great. I was thinking of putting a static method on VictoryHistogram that we could reference below instead of directly importing this function.

Reply:
Yeah, I'd definitely prefer it as a static on `VictoryHistogram`